scripts/index.js

```diff
@@ -40,9 +40,11 @@ config.output.chunkFilename = `js/[name].chunk.js`;
 
 // update media path destination
 const reactScriptsMajorVersion = getReactScriptsMajorVersion(reactScriptsVersion);
+const reactScriptsPatchVersion = getReactScriptsPatchVersion(reactScriptsVersion);
 if (reactScriptsMajorVersion >= 2) {
-  config.module.rules[3].oneOf[0].options.name = `media/[name].[hash:8].[ext]`;
-  config.module.rules[3].oneOf[7].options.name = `media/[name].[hash:8].[ext]`;
+  const oneOfIndex = reactScriptsPatchVersion >= 4 ? 2: 3;
```

Comment:
I think we have to take into consideration the minor as well otherwise on version `2.1.0` you revert back to previous index whereas you shouldn't. Something along the line should do the trick : `minor >= 1 || patch >= 4 ? 2 : 3`.